src/ccl_background.c

```diff
@@ -196,16 +218,19 @@ static int growth_ode_system(double a,const double y[],double dydt[],void *param
 INPUT: scale factor
 TASK: Define the ODE system to be solved in order to compute the growth (of the density)
 * in the case in which we use the mu / Sigma quasistatic parameterisation of modified gravity
+MI: tag to get started 
```

Comment:
Maybe this is a comment left for yourself?

Reply:
removed.